### PR TITLE
Introduce Integration Test

### PR DESCRIPTION
**Pull request by xiazhvera**
*Description of changes:*
Bring in the integration test
1. Replaced Odin Credential with AWS Secret Manager
2. Used surefire plugin to run test with different property set.
3. Added uuid for clients, and topics to avoid conflicts on test device.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: aws-iot-device-sdk-java/src/test/java/com/amazonaws/services/iot/client/system_props_websocket.properties [hunk 1]
```diff
@@ -0,0 +1,7 @@
+clientEndpoint=ajje7lpljulm4-ats.iot.us-east-1.amazonaws.com
+clientId=DefaultClientId
+thingName=DefaultThingName
+privateMaterial=arn:aws:secretsmanager:us-east-1:123124136734:secret:V1IotSdkIntegrationTestWebsocketSecretAccessKey-MKTSaV
```
Comment (TwistedTwigleg) [inline aws-iot-device-sdk-java/src/test/java/com/amazonaws/services/iot/client/system_props_websocket.properties:4]: Question: Does the `ClientEndpoint` property in this file need to be in a secret?

**Review by TwistedTwigleg — APPROVED** [review]
Looks good to me! 👍 
There's one minor thing I have a question on, but it isn't a blocker or anything.

File: aws-iot-device-sdk-java/src/test/java/com/amazonaws/services/iot/client/system_props_websocket.properties (hunk 1 above)
Comment (xiazhvera) [reply aws-iot-device-sdk-java/src/test/java/com/amazonaws/services/iot/client/system_props_websocket.properties:4]: It should be fine, we could not connect to endpoint without the correct authorization. Of cause we can always do it in a safer way and save it on the aws secret manager.

Comment (TwistedTwigleg) [reply aws-iot-device-sdk-java/src/test/java/com/amazonaws/services/iot/client/system_props_websocket.properties:4]: Ah okay, then I'd say it's fine then. I just was not sure and wanted to ask.

**Review by sbSteveK — APPROVED** [review]

**Pull request merged**